hlint/hlint.dhall

```diff
@@ -35,6 +35,21 @@ in [ rule.Arguments { arguments =
    , hintNote "pure ()" "pass" "Use 'pass'"
    , hintNote "return ()" "pass" "Use 'pass'"
 
+   -- One
+   , warnSimple "(:[])" "one"
+   , warnSimple "(Data.List.NonEmpty.:|[])" "one"
+   , warnSimple "Data.Sequence.empty" "one"
```

Comment:
I think here is a type. Should be `singleton` instead of `empty`:

* https://hackage.haskell.org/package/containers-0.6.0.1/docs/Data-Sequence.html#v:singleton